### ISSUE TITLE
Config Needs Documentation

### ISSUE THREAD

**Issue by quixQuery** [TheLMiffy1111/JAOPCA#30]
Mod works great, but it has created 3x3 blocks for materials that already have 2x2 blocks. Looked at config to see if I could disable these, but had no idea what I would change once I found the materials I was looking for. Curseforge had no link to an external wiki, and this github's wiki is in it's early stages, so I figured I would post this as an issue.

Thanks for the great mod btw.

**Comment by TheLMiffy1111** [comment]
Will do.
Put "block" without quotes in the material's blacklist to disable it.

**Comment by TheLimePixel** [comment]
How do you disable the molten versions of metals?

**Comment by TheLMiffy1111** [comment]
Put "molten" without quotes in the material's blacklist to disable it.

**Comment by TheLimePixel** [comment]
What about alchemical ore dust? I put dustAlch in the blacklist and it didn't work.

**Comment by TheLMiffy1111** [comment]
Currently JAOPCA does ore compatibility in modules for each material, because I figured that if one does not want a entry for a material, it is that they do not want the ore compatibility for the mod. If you want to disable alchemical ore dust, put "skyresources" without quotes in the material's blacklist.

I hope I made sense

**Comment by Fortist** [comment]
I have 'gear' 'rod' and 'plate' on separate lines for a few materials.

Is there a "Get all this shit for every material out" section?
I would have assumed it's the JAOPwhatever section, but that doesnt seem to work.
And for some materials it still shows up, if the blacklist is written right.
 ```
 S:moduleBlacklist <
	gear
	rod
	plate
     >

```

**Comment by Mike-U5** [comment]
So, how does one disable all the Abyssalcraft Crystal clusters?

**Comment by Silent-Hunter** [comment]
This definitely still needs some documentation. I can't figure out how to make it not override Mekanism's ability to turn sulfer dust into gunpowder using the Enrichment Chamber. It's turning it back into sulfer instead. Is there a way to blacklist this?

I've tried adding "sulfer" with and without quotes to the various blacklists, but when I reboot the server they just revert to being empty, I assume because my edit was invalid.

**Comment by TheLMiffy1111** [comment]
> This definitely still needs some documentation. I can't figure out how to make it not override Mekanism's ability to turn sulfer dust into gunpowder using the Enrichment Chamber. It's turning it back into sulfer instead. Is there a way to blacklist this?
> 
> I've tried adding "sulfer" with and without quotes to the various blacklists, but when I reboot the server they just revert to being empty, I assume because my edit was invalid.

you spelled "sulfur" wrong
but yea this might be an oversight

**Comment by Silent-Hunter** [comment]
Oh wow you're right! Now I need to see if I was spelling it wrong in the config, or if I only spelled it wrong here.

**Comment by Silent-Hunter** [comment]
Unfortunately spelling sulfur correctly still doesn't do it.

**Comment by Zafkiel204** [comment]
Similar issue is happening to me.
I want to remove the auto generated blocks added by the mod so i tried using "block" and "storage_blocks" (without quotes) in the blacklist and also used "*" in the module config for storage blocks and neither of them seems to be working.
When i launch the game it creates a bak file with the changes i made but it removes those changes from the actual config files so nothing really happens.
Tried deleting the config file to see if it would restore it from the bak file but nope, it just creates a new config file with the default settings.

Currently the only methods to remove those blocks (that i've tested) seems to be using KubeJS or Item Obliterator to forcefully remove them.

I've also removed mods to the point where i only have JAOPCA and JEI installed and this still happens.

This thread is the only "documentation" i can find about the config of this mod so im inclined to believe that im just dumb and doing something wrong instead of it being an error but it would be good to have some documentation on this regardless of what the answer to my problem is.

**Comment by TheLMiffy1111** [comment]
> Similar issue is happening to me. I want to remove the auto generated blocks added by the mod so i tried using "block" and "storage_blocks" (without quotes) in the blacklist and also used "*" in the module config for storage blocks and neither of them seems to be working. When i launch the game it creates a bak file with the changes i made but it removes those changes from the actual config files so nothing really happens. Tried deleting the config file to see if it would restore it from the bak file but nope, it just creates a new config file with the default settings.
> 
> Currently the only methods to remove those blocks (that i've tested) seems to be using KubeJS or Item Obliterator to forcefully remove them.
> 
> I've also removed mods to the point where i only have JAOPCA and JEI installed and this still happens.
> 
> This thread is the only "documentation" i can find about the config of this mod so im inclined to believe that im just dumb and doing something wrong instead of it being an error but it would be good to have some documentation on this regardless of what the answer to my problem is.

you need quotes now

**Comment by Zafkiel204** [comment]
Yep, that worked. Guess thats what happens when i try to follow a comment from 2017

**Comment by Lucksinia** [comment]
Asking here, because it's a general documentation tread, apparently. (why, by the way?)
Is there a way to force a module to work without a specific mode to generate it from? (whitelist?)
For example, add iron dust into the word?
From what I can see scouring configs, just adding "iron" to the Materials override section in main config file, does nothing...
